examples/sign-leaf-with-ca.rs

```diff
@@ -0,0 +1,33 @@
+use rcgen::{BasicConstraints, Certificate, CertificateParams, IsCa};
+use std::fs;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+	let mut ca_params: CertificateParams = Default::default();
```

Comment:
I think the idiomatic way to write these would be `CertificateParams::default()`.